core/src/main/java/org/elasticsearch/action/admin/cluster/stats/TransportClusterStatsAction.java

```diff
@@ -92,7 +92,7 @@ protected ClusterStatsNodeResponse newNodeResponse() {
 
     @Override
     protected ClusterStatsNodeResponse nodeOperation(ClusterStatsNodeRequest nodeRequest) {
-        NodeInfo nodeInfo = nodeService.info(false, true, false, true, false, true, false, true, false);
+        NodeInfo nodeInfo = nodeService.info(false, true, false, true, false, true, false, true, false, true);
```

Comment:
given all the falses here, do we want to default to false too? IndexingBuffer seems too low level for ClusterStats